zzzz_modular_occulus/code/modules/surgery/organic_damage.dm

```diff
@@ -0,0 +1,63 @@
+/datum/surgery_step/fix_brute
+	allowed_tools = list(
+		/obj/item/stack/medical/advanced/bruise_pack = 100,
+		/obj/item/stack/medical/bruise_pack = 20,
```

Comment:
Non-blocking; what does the number for the tool mean? Is that an analogue for the tool quality?

Reply:
Yes. As bruise packs don't have that quality.